vizro-core/src/vizro/_vizro.py

```diff
@@ -76,6 +76,18 @@ def _pre_build():
             if hasattr(model, "pre_build"):
                 model.pre_build()
 
+    @staticmethod
+    def _clear_state():
```

Comment:
Hm, if we only use `clear` it may be too linked to the `Vizro` class no? Ultimately I think it's ok to be attached to it, because it thematically belongs to it, but if people then type `Vizro._clear` they could think that the `Vizro` class is just some data structure. This clear method somewhat transcends that. But honestly not sure...

Reply:
I was thinking mostly about consistency but I definitely see your point. I don't have a strong preference on this one to be honest, so either way would be fine for me :)

Reply:
Let's leave for now unless someone else has a stronger opinion